### DISCUSSION TITLE
Any way to display both line number and relative line number in 2 gutters?

### DISCUSSION THREAD

**Discussion by jerabaul29** [helix-editor/helix#6990] [General]
Is there any way to display both line number and relative line number in 2 gutters, instead of either or? If so, what is the config snippet? If no, any way this could be added? :)

**Comment by Pieter-JanLavaerts** [comment]
bump, I'm also looking if this is possible

**Comment by sommerper** [comment]
That would actually be nice. I often pair program over zoom and it's easier when people can just say which line number I should pay attention to. But at the same time I use the relative numbers to navigate.
